pipelines/base/peribolos-dump-config.yaml

```diff
@@ -1,23 +1,41 @@
 apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
-  name: dump-config
+  name: peribolos-dump-config
 spec:
   params:
   - name: REPO_NAME
     type: string
     default: .github
   - name: INSTALLATION_ID
     type: string
-  - name: GIT_EMAIL
-    type: string
-  - name: GIT_NAME
-    type: string
   steps:
-    - name: dump-and-push-config
+    - name: dump-config
+      image: quay.io/skopecky/peribolos-fix
+      volumeMounts:
+        - mountPath: /mnt/shared
+          name: shared-data
+        - mountPath: /mnt/secret
+          name: github-token
+      env:
+        - name: ORG_NAME
+          valueFrom:
+            secretKeyRef:
+              name: peribolos-$(params.INSTALLATION_ID)
+              key: orgName
+      script: |
+        #!/bin/sh
+
+        # A script is needed to redirect the output to a file
+        echo "Dumping org config ..."
```

Comment:
No need for logging here, peribolos logs enough by itself. Peribolos also logs in easily parseable json format, which would interfere with this line down the road if we try to use the logs in Loki.